src/lib/components/Tree/TreeContainer.svelte

```diff
@@ -56,9 +58,10 @@
 		/>
 	{/key}
 
-	{#if environment.current.isStandalone}
+	{#if environment.current.isStandalone && partID.current}
```

Comment:
Drive by: don't show this UI if we're not connected to a robot and instead just drawing arbitrarily